Makefile

```diff
@@ -24,6 +24,12 @@ lint: golangci-lint ## Run golangci-lint linter
 
 .PHONY: e2e
 # export GITHUB_AUTH_TOKEN with  personal access token to run the e2e
-e2e:
-	ginkgo  -v -cover  ./...
+e2e: build
+	ginkgo --skip="E2E TEST:executable" -p  -v -cover  ./...
 
+
+.PHONY: ci-e2e
+# export GITHUB_AUTH_TOKEN with  personal access token to run the e2e
```

Comment:
Instead of comment, why not check GITHUB_AUTH_TOKEN and fail loudly if not set.

Reply:
Took care of it,